rules/cloud/aws/aws_ses_messaging_enabled.yml

```
@@ -0,0 +1,26 @@
+title: Configure AWS SES To Send Messages
+id: 60b84424-a724-4502-bd0d-cc676e1bc90e
+status: experimental
+description: Identifies email sending feature is enabled for an account and the email address verification request is dispatched
+references:
+    - https://unit42.paloaltonetworks.com/compromised-cloud-compute-credentials/
+author: Janantha Marasinghe
+date: 2022/12/12
+tags:
+    - attack.t1583.006
+    - attack.resource_development
+logsource:
+    product: aws
+    service: cloudtrail
+detection:
+    selection1:
+        eventSource: ses.amazonaws.com
+        eventName: 'UpdateAccountSendingEnabled'
+    selection2:
+        eventSource: ses.amazonaws.com
+        eventName: 'VerifyEmailIdentity'
+    timeframe: 5m
+    condition: selection1 and selection2
```

- These can be merged into one selection.
```suggestion
    selection:
        eventSource: ses.amazonaws.com
        eventName:
            - 'UpdateAccountSendingEnabled'
            - 'VerifyEmailIdentity'
    timeframe: 5m
    condition: selection
```

Also, can you provide more context to suspicious usage? the paloalto blod doesn't go into much details.

---

Based on the PA writeup the adversary tries to access different services within AWS to understand which service they have privileges to do things. In that specific one they found they have access to SES. They first enable the account to have the ability to message and then also prior to being able to send e-mail they verify it and subsequently launches a phishing attack.